src/main/java/org/candlepin/subscriptions/product/UpstreamProductData.java

```diff
@@ -63,6 +63,8 @@ private enum Attr {
     IFL,
     SOCKET_LIMIT,
     SERVICE_TYPE,
+    LEVEL_1,
```

Comment:
Not sure if we need '_'  but, to be consistent may be we can add it to offering table columns?